esmvalcore/preprocessor/_multimodel.py

```diff
@@ -439,9 +486,9 @@ def multi_model_statistics(products,
     workflow and provenance information, and this option should typically be
     ignored.
 
-    Apart from the time coordinate, cubes must have consistent shapes. There
-    are two options to combine time coordinates of different lengths, see the
-    ``span`` argument.
+    Cubes must have consistent shapes apart from a potential time dimensions.
```

Comment:
```suggestion
    Cubes must have consistent shapes apart from a potential time dimension.
```
from a time dimension 
or 
from dimensions